app/assets/javascripts/comments.coffee

```diff
@@ -0,0 +1,20 @@
+class Comment
+  constructor: ->
+    @bindEvents()
+
+  bindEvents: =>
+    $(document).on "ajax:success", "[data-behavior='comment_destroy']",
```

Comment:
also, you have only one form on page, so you can use class name and get rid of data attribute